src/main.rs

```diff
@@ -30,39 +30,48 @@ fn run_command(command: &str, work_dir: &Parent) -> Output {
         )
 }
 
-fn read_file(f: &FileArg) -> Result<String, std::io::Error> {
+fn die<A>(msg: String) -> A {
```

Comment:
would it be possible to support the same function signature as `format!()` so the message and additional parameters can be passed at once?

Reply:
From my experience, `format!` needs a string literal in the first place, it can’t take an argument, because it’s compiled as a macro.

Reply:
ok, let's leave it like that then